ansible/roles/snactor/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+snactor:
+    copr_repo: "evilissimo/leapp"
+    git_repo: https://github.com/leapp-to/snactor.git
+    git_dest: "{{ common.workdir }}/snactor/"
+    packages:
```

Comment:
IMO this should be replaced by shell/command (dnf builddeps) because we are duplicating the data here.